src/WebJobs.Script/Diagnostics/Extensions/ExtensionBundleLoggerExtension.cs

```diff
@@ -13,7 +13,7 @@ internal static class ExtensionBundleLoggerExtension
 
         private static readonly Action<ILogger, string, Exception> _contentProviderNotConfigured =
             LoggerMessage.Define<string>(
-            LogLevel.Information,
```

Comment:
@soninaren - Changing logs from Extension bundles to debug level.  For system logs default log level is Debug. We will continue to see these logs on Kusto. Can you please ACK?

Reply:
Will these logs appear in the core tools? We should be good for this particular message.